lib/matplotlib/colors.py

```diff
@@ -1139,61 +1139,67 @@ class DivergingNorm(TwoSlopeNorm):
     ...
 
 
+def _make_norm_from_scale(scale_cls, base_cls=None, *, init=None):
+    if base_cls is None:
+        return functools.partial(_make_norm_from_scale, scale_cls, init=init)
+
+    if init is None:
+        init = lambda vmin=None, vmax=None, clip=False: None
```

Comment:
the signature overriding mess is because the signatures of SymLogScale and SymLogNorm are not consistent with one another either...